squareconnect/api_client.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, host=None, header_name=None, header_value=None, cookie=None):
             self.host = host
         self.cookie = cookie
         # Set default User-Agent.
-        self.user_agent = 'Square-Connect-Python/2.0.2'
+        self.user_agent = 'Swagger-Codegen/2.0.2/python'
```

Comment:
Ugh, its annoying that this is changing and all the templates use different conventions. I'm going to ask someone if there is a group who can decide on a standard for all codegen languages.